openeogeotrellis/configparams.py

```diff
@@ -28,6 +28,13 @@ def __init__(self, env=os.environ):
             "epod-master1.vgt.vito.be:6668,epod-master2.vgt.vito.be:6668,epod-master3.vgt.vito.be:6668",
         )
 
+        self.yarn_rest_api_base_url = env.get(
+            "YARN_REST_API_BASE_URL", "https://epod-master1.vgt.vito.be:8090"
+        )
+        self.yarn_auth_use_kerberos = smart_bool(
+            env.get("YARN_AUTH_USE_KERBEROS", "yes")
+        )
```

Comment:
Do we need this `yarn_auth_use_kerberos` config param? AFAIK we always require Kerberos authentication to access YARN.

Reply:
Good question.
The actual purpose here was to be able to turn Kerberos authentication off during unit tests. However that does not really mean that we need to be able to configure this for a production setup.

I will try to find a better solution. That may indeed simplify a few things, as your other comments below also indicated.